main/daily_update.py

```diff
@@ -175,9 +178,9 @@ def extract_journey_file(sf_api, sqlfile_name, platform_name):
     :param platform_name: salesforce or pardot
     :return: dataframe
     """
-    fd = open(os.path.abspath(os.path.join(os.pardir, 'sqlserver', sqlfile_name)), 'r')
+    fd = open(os.path.join(settings.path, 'sqlserver/{}'.format(sqlfile_name)), 'r')
     if platform_name == 'salesforce':
-        sqlfile = fd.read().format(settings.OwnerId, settings.PardotUser)
+        sqlfile = fd.read().format(settings.OwnerId, settings.PardotUser, settings.SovereignUser)
```

Comment:
what is SOVEREIGN user?

Reply:
It's Sovereign CRM. There are some participant journeys whose owner is Sovereign CRM.